packages/suite/src/components/suite/modals/Pin/messages.ts

```diff
@@ -0,0 +1,22 @@
+import { defineMessages } from 'react-intl';
+
+const definedMessages = defineMessages({
+    TR_ENTER_PIN: {
+        id: 'TR_ENTER_PIN',
+        defaultMessage: 'Enter {device.label} PIN'
+    },
+    TR_PIN_DISPLAY: {
+        id: 'TR_PIN_DISPLAY',
+        defaultMessage: 'The PIN layout is displayed on your Trezor.',
+    },
+    TR_HOW_PIN_WORKS: {
+        id: 'TR_HOW_PIN_WORKS',
+        defaultMessage: 'Not sure how PIN works?',
+    },
+    TR_LEARN_MORE: {
```

Comment:
TR_LEARN_MORE is already defined in this file 
https://github.com/trezor/trezor-suite/blob/c2d5367e2e8074e15910e12787d8d3b556e70768/packages/suite/src/components/suite/modals/messages.ts#L16-L19

We can't have duplicate message IDs otherwise the extraction of messages will fail. So instead you can just import it from the file linked above.